client.go

```diff
@@ -45,10 +33,6 @@ type (
 	}
 )
 
-func (resp *Response) IsUnauthorized() bool {
-	return resp.StatusCode() == http.StatusUnauthorized
-}
-
 type reqConfig struct {
```

Comment:
can we move this type into the group of types in the top section?

Reply:
yep